website/blog/2018-06-25-on-consuming-and-publishing-es6+-packages.md

```diff
@@ -0,0 +1,218 @@
+
+## What Then?
+
+The next step is to start thinking "how do we actually make this happen"?
+
+Is it as straightforward as just running Babel over `node_modules` with the same configuration for your application?
+
+## Caveats
+
+### Using Non-Standard Syntax
+
+This is what Dan was warning about. There are many issues with *shipping* uncompiled proposal syntax.
+
+#### Using Proposals
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="und" dir="ltr"><a href="https://t.co/femUb4vgxh">pic.twitter.com/femUb4vgxh</a></p>&mdash; Rach Smith 🌈 (@rachsmithtweets) <a href="https://twitter.com/rachsmithtweets/status/892478598765887488?ref_src=twsrc%5Etfw">August 1, 2017</a></blockquote>
+
+We already recommend that people should be careful when using proposals lower than Stage 3, let alone publishing them.
+
+But telling people not to use Stage X goes against the whole purpose of Babel in the first place. A big reason why proposals gain improvements and move forward are because of the feedback the committee gets from real-world usage (whether in production or not) based on using it via Babel.
+
+There is certainly a balance to be had here. We don't want to scare people away from using new syntax (that is a hard sell 😂), but we also don't want people to get the idea that "once it's in Babel, the syntax is official or immutable". Ideally people look into the purpose of a proposal and make the tradeoffs for their use case.
+
+One of the most common things people do is use the Stage 0 or Stage 2 presets. We plan to remove the stage presets in v7 after much back and forth. I thought at first it would be convienent, that people would make their own unofficial ones anyway, or it might help with "JavaScript fatigue". I find now that it just causes more of an issue: people continue to copy/paste configs with out understanding what goes into a preset in the first place. After all, seeing `"stage-0"` says nothing. My hope is that in making the decision to use proposal plugins explicit, people will have to learn what non-standard syntax they are opting into.
+
+#### Staging Process
+
+- The [TC39 staging process](https://tc39.github.io/process-document/) does not always move forward: there is the possibility for a proposal to move to any point in the process: even moving backwards from Stage 3 to Stage 2 as the case with Numeric Separators (`1_000`) or dropped entirely (`Object.observe()`, and others you may have forgotten)
+- It can be hard to differentitate between the stages if you aren't involved: most people would advise that Stage 0-2 is unstable though.
+- Summary: Stage 0 has no criteria and is just an idea, Stage 1 is accepting that the problem is worth solving, Stage 2 is about describing a solution in spec text, Stage 3 means the specific solution is thought out, and Stage 4 means that it is ready for inclusion with tests, multiple browser implementations, and in-the-field experience.
+
+#### Publishing Non-standard Syntax
```

Comment:
This may be a new `##`-level section. Now we're out of "problems with compiling", and in to "problems when code with assumptions in them are published without a way for consumers to easily know or deal with what those assumptions were".

Reply:
Yeah I guess I'm combining "compiling" for both the user and author side which might be confusing. Yeah the audience is all developers even if they aren't a library author because they should understand on the library side why this isn't a thing yet?